yt/loaders.py

```diff
@@ -90,6 +90,8 @@ def load(
     yt.utilities.exceptions.YTAmbiguousDataType
         If the data format matches more than one class of similar specialization levels.
     """
+    from yt.frontends import _all  # type: ignore [attr-defined] # noqa
```

Comment:
Oh and one extra comment/question -- by nesting this import in the `load` and `load_simulation` calls, does that mean one could `import yt` and then directly instantiate a frontend without ever importing all the other frontends? That would actually be a pretty nice consequence beyond simply delaying the imports... might be worth a mention in the docs even. Could make a big difference for batch processing scripts.

Reply:
indeed, I think with this patch you'd be able to short-circuit all frontends being loaded, but I think at this point it should be seen as an implementation detail rather than a feature (in particular because circumventing `yt.load` would require different code if your frontends isn't builtin, but lives in an extension)